src/main/java/com/opencart/pages/AdminDashboard.java

```diff
@@ -0,0 +1,76 @@
+package com.opencart.pages;
+
+import com.opencart.containers.CurrencyContainerAdmin;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class AdminDashboard extends BasePage {
```

Comment:
I think it is better to rename it to CurrencyAdminPage.
AdminDashboard page is a default page for "admin" panel, the same as Main page for "store".
So, that means that AdminDashboard page opens firstly when your login to "admin" panel.
Then you need to navigate to AdminCurrency page from there. For that, describe the left-navigation panel elements and business logic within separate classes, then just aggregate that logic to AdminDashboard (as we have done with HeaderPage).